lib/Auth/Process/COmanageDbClient.php

```diff
@@ -163,77 +164,32 @@ class sspmod_attrauthcomanage_Auth_Process_COmanageDbClient extends SimpleSAML_A
         . ' AND cert.cert_id IS NULL'
         . ' AND NOT cert.deleted';
 
-      private $couQuery = 'SELECT'
-        . ' cou.name AS cou_name,'
-        . ' avg(DISTINCT cou.id)::INT AS cou_id,'
-        . ' string_agg(DISTINCT role.title, \',\') AS title,'
-        . ' string_agg(DISTINCT role.affiliation, \',\') AS affiliation,'
-        . ' string_agg(DISTINCT cast(members.member AS text), \',\') AS member,'
-        . ' string_agg(DISTINCT cast(members.owner AS text), \',\') AS owner'
-        . ' FROM cm_cous AS cou'
-        . ' INNER JOIN cm_co_person_roles AS ROLE ON cou.id = role.cou_id'
-        . ' AND role.co_person_role_id IS NULL'
-        . ' AND role.status = \'A\''
-        . ' AND NOT role.deleted'
-        . ' AND role.co_person_id = :coPersonId'
-        . ' AND NOT cou.deleted'
-        . ' INNER JOIN cm_co_groups AS groups ON groups.cou_id = cou.id'
-        . ' AND groups.group_type = \'A\''
-        . ' AND NOT groups.deleted'
-        . ' AND groups.co_group_id IS NULL'
-        . ' LEFT OUTER JOIN cm_co_group_members AS members ON members.co_group_id = groups.id'
-        . ' AND members.co_person_id = role.co_person_id'
-        . ' AND NOT members.deleted'
-        . ' AND members.co_group_member_id IS NULL'
-        . ' GROUP BY cou_name'
-        . ' ORDER BY cou_name DESC';
-
-    private $groupQuery = 'SELECT'
-        . ' DISTINCT (gr.name),'
-        . ' gm.member,'
-        . ' gm.owner'
-        . ' FROM cm_co_groups AS gr'
-        . ' INNER JOIN cm_co_group_members AS gm'
-        . ' ON gr.id=gm.co_group_id'
-        . ' WHERE'
-        . ' gm.co_person_id= :coPersonId'
-        . ' AND gm.co_group_member_id IS NULL'
-        . ' AND NOT gm.deleted'
-        . ' AND gr.co_group_id IS NULL'
-        . ' AND NOT gr.deleted'
-        . ' AND gr.co_id = :coId'
-        . ' AND gr.group_type = \'S\''
-        . ' AND gr.status = \'A\''
-        . ' ORDER BY'
-        . ' gr.name DESC';
-
-    public function __construct($config, $reserved)
-    {
+    public function __construct($config, $reserved) {
```

Comment:
Opening braces for methods MUST go on the next line, and closing braces MUST go on the next line after the body. (see https://www.php-fig.org/psr/psr-2/#1-overview)